### ISSUE TITLE
Resend and Delete buttons missing / not working on error store page

### ISSUE THREAD

**Issue by jacodg** [frankframework/frankframework#5594]
**Describe the issue**
With latest 7.9 timestamp version I cannot resend my message in the error store. The Resend and Delete buttons were missing but after some browsing and shift-reload they appeared but they are not working.

With 7.8.3 this is working without a problem.

**Reporter**
Jaco de Groot

**To Reproduce**
1. Run Frank2Example3 in/with the Frank!Runner
2. Read the descriptions of the adapters and follow the instructions to get a message in the error store
3. See that you cannot Resend the message
4. Do the same with 7.8.3
5. See that is it working without a problem

**Screenshots**
\-

**Configuration**
See Frank2Example3 

**Input**
Random string

**Environment**
```
FF! 7.9-20231018.223428: Frank2Example3
Running on xxx using Apache Tomcat/9.0.74
Java Version: OpenJDK Runtime Environment (1.8.0_292-b10)
Heap size: 297M, total JVM memory: 608M
Free disk space: 48GB, total disk space: 475GB
Up since: 2023-10-19 16:58:37 (16m) 
```

**Additional Environment**
 - Browser: Firefox

**Additional Context**
\-

**Comment by jacodg** [comment]
Is doesn't seem to be working. With 8.1.0-20240115.192321 I see:

![image](https://github.com/frankframework/frankframework/assets/2970084/72cbdf1a-c86a-444d-89b5-c8caa38d7406)

And on selecting the message and using Resend Selected button:

![image](https://github.com/frankframework/frankframework/assets/2970084/e6ed696a-d889-495e-b0b9-f2fffee39047)

**Comment by Matthbo** [comment]
Fixed in #6130, #6131, #6132
